docs/source/resources/package-spec.rst

```diff
@@ -263,7 +263,7 @@ the command line with ``conda install``, such as
 ``conda install python=3.4``. Internally, conda translates the
 command line syntax to the spec defined in this section.
 
-EXAMPLE: python=3.4 is translated to python 3.4*.
+EXAMPLE: python=3.4 is translated to python 3.4.*. ``conda search 'python=3.1'`` does NOT bring up Python 3.10, only Python 3.1.*.
```

Comment:
the error is pertaining a special character. i believe enveloping this in a doublew quote will be needed.. 
<img width="1077" alt="image" src="https://github.com/conda/conda-build/assets/110486050/3b2aaffb-7e76-40c9-b42b-f4200bf2bf9e">

Reply:
Double quotes caused that error that you screenshotted in the first place. I'm definitely a little confused. Have quotes always been a problem here?

Reply:
`conda install` appears to work just fine with single or double quotes for me, but not `conda search`.

Reply:
Okay, it seems my conda install was broken. I've uninstalled and reinstalled and am no longer getting those errors! Apologies for that. I'll resolve this convo.